src/widgets/shared-utils.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
+
+export const openWorkspaceTab = (componentToAdd, componentName, params?) => {
```

Comment:
This is the same as beforehand, right? Maybe just fix the other branch and update this PR. Then everything should be fine!

Reply:
Yeah, let me close this one since the changes have already been merged in another PR.